mmdet/datasets/pipelines/loading.py

```diff
@@ -11,19 +11,30 @@
 @PIPELINES.register_module()
 class LoadImageFromFile(object):
 
-    def __init__(self, to_float32=False, color_type='color'):
+    def __init__(self,
+                 to_float32=False,
+                 color_type='color',
+                 file_client_args=dict(backend='disk')):
         self.to_float32 = to_float32
         self.color_type = color_type
+        self.file_client_args = file_client_args.copy()
```

Comment:
How come we don't instantiate `self.file_client` at __init__ so that `self.file_client_args` can be removed?

Reply:
If we init a FileClient in the `__init__()` method, an error will be raised.